src/display/canvas.js

```diff
@@ -1347,7 +1347,8 @@ var CanvasGraphics = (function CanvasGraphicsClosure() {
       this.moveText(0, this.current.leading);
     },
 
-    paintChar: function CanvasGraphics_paintChar(character, x, y) {
+    paintChar: function CanvasGraphics_paintChar(character, x, y,
```

Comment:
Nit: Taking full advantage of ES6 syntax, the entire method signature can be simplified to:
```javascript
paintChar(character, x, y, patternTransform) {
```